components/base/Table.tsx

```diff
@@ -35,6 +41,10 @@ const StyledTable = styled.div`
     }
     :nth-last-of-type(1) {
       border-right: 1px solid ${GREY_2};
+      /* This ensures the last column width doesn't shrink to a point where the text wraps around each word */
```

Comment:
Added a min-width on the last column to prevent this:
<img width="284" alt="Screen Shot 2020-06-11 at 2 36 42 PM" src="https://user-images.githubusercontent.com/44006085/84431517-08b88580-abf1-11ea-92a4-ce735ee2be9a.png">